common/frame.go

```diff
@@ -567,7 +567,7 @@ func (f *Frame) waitForFunction(
 
 	execCtx := f.executionContexts[world]
 	if execCtx == nil {
-		return nil, fmt.Errorf("cannot find execution context: %q", world)
+		return nil, fmt.Errorf("execution context %q not found", world)
```

Comment:
"Execution context" is a CDP concept, so yes, it's confusing to users, but "function not found on the frame" is an incorrect message in this case. As a user, I'd rather see confusing than wrong error messages.

Since we'll have to agree on the right approach for this, let's discuss it in a new issue.